Gemfile

```diff
@@ -7,7 +7,7 @@ gemspec
 CURRENT_RAILS_VERSION = "7.1"
 rails_version = ENV.fetch("RAILS_VERSION", CURRENT_RAILS_VERSION)
 
-gem "minitest"
+gem "minitest", "< 5.25.0" # minitest 5.25.0+ is incompatible with minitest-hooks
```

Comment:
I think this can be removed https://github.com/minitest/minitest/commit/6d838438ee53897df20b5aef3df35fc291018b9e

Reply:
Sure, I'll clean up this branch quite a lot, but I needed to make sure the tests weren't failing.